pkg/config/config.go

```diff
@@ -58,11 +62,13 @@ func ToController(s *Serialized, cfg *Controller) (*Controller, error) {
 	if cfg == nil {
 		cfg = &Controller{}
 	}
+
 	cfg.Report = s.Report
 	cfg.StoragePath = s.StoragePath
 	cfg.Endpoint = s.Endpoint
 	cfg.Impersonate = s.Impersonate
 	cfg.Gather = s.Gather
+	cfg.EnableGlobalObfuscation = cfg.EnableGlobalObfuscation || s.EnableGlobalObfuscation
```

Comment:
I guess `cfg.EnableGlobalObfuscation` will be always false so probably not necessary to add it here.

Reply:
Does it work now the way that it would be enabled if enableGlobalObfuscation is true in either the config or the secret?

Reply:
The config file is a part of the image (and we don't expect users building their custom IO images) so this is useful only for dev purposes IMO.

Reply:
Yeah, it doesn't make sense here, we only need it in merge config to be able to enable obfuscation by secret or by config.